[Bug] Compilation failure with ReflectMut::Function when 'reflect_functions' Bevy feature is enabled

I've encountered a compilation error when using bevy_ecs_tiled in a project where the underlying bevy dependency has the reflect_functions feature enabled (which adds the ReflectMut::Function variant).

The current implementation of the match statement at load.rs is non-exhaustive when this Bevy feature is active, causing a compile failure.

Error Details:

```shell
error[E0004]: non-exhaustive patterns: `ReflectMut::Function(_)` not covered

   --> bevy_ecs_tiled-0.8.2\src\tiled\properties\load.rs:600:11

    |

600 |     match object.reflect_mut() {

    |           ^^^^^^^^^^^^^^^^^^^^ pattern `ReflectMut::Function(_)` not covered

    |
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 ## [unreleased]
 
+### Bugfixes
+
+- Fix compilation failure with `ReflectMut::Function` when `reflect_functions` Bevy feature is enabled (#159)
+
 ## v0.9.5
 
 ### Features

--- a/src/tiled/properties/load.rs
+++ b/src/tiled/properties/load.rs
@@ -649,7 +649,7 @@ fn hydrate(object: &mut dyn PartialReflect, obj_entity_map: &HashMap<u32, Entity
         // Cannot hydrate a Set since it does not have a get_mut() function
         ReflectMut::Set(_) => {}
         // we don't care about any of the other values
-        ReflectMut::Opaque(_) => {}
+        _ => {}
     }
 }
 

